pkg/build/build_implementation.go

```diff
@@ -148,6 +167,7 @@ func (bc *Context) buildImage(ctx context.Context) error {
 		}
 	}
 
+	// TODO: What about accounts ?
```

Comment:
For MVP I would assume we inherit it from the base image.

A slightly better option would be to give the possibility to override all the accounts.

Giving to much flexibility by allowing both, adding and overwriting will make API more complex - we should consider if it's worth it.